Core.lua

```diff
@@ -878,7 +878,9 @@ function BigWigs:RegisterModule(name, module)
 	if module:IsBossModule() and module.toggleoptions then
 		opts = {}
 		for _, v in pairs(module.toggleoptions) do
-			if v ~= -1 then
+			if module.defaultDB and module.defaultDB[v] ~= nil then
```

Comment:
realized you didn't have this (maybe from balake?), can default certain alerts to off